web/samples/test-filament-viewer.html

```diff
@@ -83,7 +83,8 @@ <h3>Skybox, fixed size</h3>
 <h3>No skybox, no IBL, fixed size, round border</h3>
 <p>
 <filament-viewer
-    src="https://prideout.net/models/dino/trex.gltf"
+    src="https://prideout.net/models/MaterialsVariantsShoe.glb"
```

Comment:
We should prob not rely on your domain name btw.

Reply:
Maybe we should make an assets repo at some point.  I tried replacing these with blob URL's from the Khronos repo and ran into fun CORS errors:

```
Access to fetch at 'https://github.com/KhronosGroup/glTF-Sample-Models/raw/master/2.0/MaterialsVariantsShoe/glTF-Binary/MaterialsVariantsShoe.glb' from origin 'http://127.0.0.1:8080' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'https://render.githubusercontent.com' that is not equal to the supplied origin. Have the server send the header with a valid value, or, if an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```